<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>584</width>
    <height>301</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Input Layers for Kharif Model</string>
  </property>
  <widget class="QPushButton" name="watershed_layer_browse">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>38</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="watershed_layer_filename">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>471</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>145</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Mini-Watershed Vector Layer :</string>
   </property>
  </widget>
  <widget class="QPushButton" name="soil_layer_browse">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>97</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>118</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Soil Cover Vector Layer :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="soil_layer_filename">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>98</y>
     <width>471</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="lulc_layer_browse">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>160</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>175</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Land-Use-Land-Cover Vector Layer :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lulc_layer_filename">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>471</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="slope_layer_browse">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>220</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>98</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Slope Raster Layer :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="slope_layer_filename">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>471</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>260</y>
     <width>156</width>
     <height>23</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>271</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>271</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
